### ISSUE TITLE
"Watched" or "Starred" have more repos than "All"

### ISSUE THREAD

**Issue by hg42** [slapperwan/gh4a#143]
"All" seems to be restricted to my own repos.

**Comment by CharString** [comment]
I agree. Current "all" is a useful filter, but should be called "mine" in addition to a default "all" option, that really shows *all* repos. Then the filter action really filters, right now it didn't occur to me to press the filter action to get to my starred repos; actually this exact issue put me on track that I could get to my starred repos thru the filter. :-)
